Dockerfile

```diff
@@ -0,0 +1,20 @@
+
+FROM python:3.7.6
+
+ENV SRC_DIR /Users/alinahazirah/Desktop/mercari_projects/mercari-build-training-2022/python

```

Comment:
if possible use relative path like `./python` instead of full path. this path most likely is not on other machine path.

Reply:
Alright, I'll change the path. Thank you for pointing it out.